app/controllers/api/v1/search_controller.rb

```diff
@@ -21,11 +21,29 @@ def index
           tags: params[:tags],
           zipcode: params[:location],
           page: params[:page],
-          per_page: params[:per_page]
+          per_page: params[:per_page],
+          languages: params[:languages]
         ).search.load&.objects
         
         generate_pagination_headers(locations)
 
+        languages = LocationsSearch.new(
+          accessibility: params[:accessibility],
+          category_ids: params[:categories],
+          distance: params[:distance],
+          keywords: params[:keyword],
+          lat: @lat,
+          long: @lon,
+          org_name: params[:org_name],
+          tags: params[:tags],
+          zipcode: params[:location],
+          page: params[:page],
+          per_page: params[:per_page],
+          languages: params[:languages]
+        ).languages
+
+        Rails.logger.info languages
```

Comment:
```suggestion
```

I think this is for debugging purposes and we can remove it?

Reply:
definitely. removed